jest.config.js

```diff
@@ -14,7 +14,6 @@ module.exports = {
     '^.+\\.svg$': 'jest-transform-stub'
   },
   setupFilesAfterEnv: ['<rootDir>/packages/testSetup.ts'],
-  transformIgnorePatterns: ['node_modules/(?!@patternfly|@novnc|@popperjs|lodash|monaco-editor|react-monaco-editor)'],
```

Comment:
I've removed `transformIgnorePatterns` as the default already excludes the `node_modules` directory. It doesn't seem that any of the listed dependencies need to be included, the tests work fine without. Aside from that this also improves the startup time for test execution by about 5 seconds on my machine.

Reply:
Hmmm I'd tried removing that option before and run into issues because of imports that we did need to transform, but if we're not seeing problems related to that when the unit tests run maybe we've resolved that issue in other ways.

If so that's AWESOME, if not we can always revert this change if need be, it's not like it will impact consumers even if it doesn't work out.